rosa_learning/deploying_application_workshop/learning-lab-overview.adoc

```diff
@@ -0,0 +1,301 @@
+* Use AWS Controllers for Kubernetes (ACK) to deploy and use an S3 bucket
+
+.Prerequisites
+
+* A provisioned ROSA cluster
+* The xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[OpenShift command line interface (CLI)]
+* A link:https://github.com/signup[GitHub account]
+
+== About the OSToy application
+
+OSToy is a Node.js application that deploys to a ROSA cluster to help explore the functionality of Kubernetes.
+
+This application has a user interface where you can:
+
+* Write messages to the log (stdout / stderr)
+* Intentionally crash the application to view self-healing
+* Toggle a liveness probe and monitor OpenShift behavior
+* Read ConfigMaps, secrets, and environment variables
+* Read and write files when connected to shared storage
+* Check network connectivity, intra-cluster DNS, and intra-communication with the included microservice
+* Increase the load to view automatic scaling of the pods by using the Horizontal Pod Autoscaler
+* Connect to an AWS S3 bucket to read and write objects
+
+=== OSToy Application Diagram
+
+image::ostoy-arch.png[OSToy architecture diagram]
+
+=== Understanding the OSToy UI
+
+image::ostoy-homepage.png[Preview of the OSToy homepage]
+
+. Pod name
+. *Home:* Application home page
+. *Persistent Storage:* Writes data to the persistent volume bound to the application
+. *Config Maps:*  Shows ConfigMaps available to the application and the key:value pairs
+. *Secrets:* Shows secrets available to the application and the key:value pairs
+. *ENV Variables:* Shows environment variables available to the application
+. *Networking:* Networking tools
+. *Pod Auto Scaling:* Increase the load of the pods and test the Horizontal Pod Autoscaler (HPA)
+. *ACK S3:* Integrate with AWS S3 to read and write objects to a bucket 
++
+[NOTE]
+====
+You must complete the ACK section of this workshop to see the "ACK S3" section of OSToy. If you do not complete that section, the OSToy application still functions.
+====
++
+. *About:* Application information
+
+=== Lab resources
+
+* link:https://github.com/openshift-cs/ostoy[OSToy application source code]
+* link:https://quay.io/ostoylab/ostoy-frontend[OSToy front-end container image]
+* link:https://quay.io/ostoylab/ostoy-microservice[OSToy microservice container image]
+* Deployment Definition YAML files:
++
+.`ostoy-frontend-deployment.yaml`
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ostoy-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ostoy-frontend
+  labels:
+    app: ostoy
+spec:
+  selector:
+    matchLabels:
+      app: ostoy-frontend
+  strategy:
+    type: Recreate
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: ostoy-frontend
+    spec:
+      # Uncomment to use with ACK portion of the workshop
```

Comment:
Not sure if I have seen like these "uncomment" callouts presented in a .yaml file, but may be fine...wasn't sure.